src/Modules/CalcOffence.lua

```diff
@@ -3393,7 +3393,14 @@ function calcs.offence(env, actor, activeSkill)
 		end
 
 		-- Calculate non-damaging ailments effect and duration modifiers
-		local enemyThreshold = enemyDB:Sum("BASE", nil, "AilmentThreshold") * enemyDB:More(nil, "Life")
+		local isBoss = env.configInput["enemyIsBoss"] ~= "None"
+		local enemyLevel = m_max(env.enemyLevel, 66)
+		local enemyBaseLife = data.monsterLifeTable[enemyLevel]
+		local enemyMapLifeMult = isBoss and data.mapLevelBossLifeMult[enemyLevel] or data.mapLevelLifeMult[enemyLevel]
+		local enemyMapAilmentMult = isBoss and data.mapLevelBossAilmentMult[enemyLevel] or 1
+		local enemyTypeMult = isBoss and 7.68 or 1
+		local enemyThreshold = enemyBaseLife * enemyTypeMult * enemyMapLifeMult * enemyMapAilmentMult * enemyDB:More(nil, "AilmentThreshold")
```

Comment:
Technically this throws away any future mods we make to enemy life (e.g. adding the life/es map mods to the config section for map mods)

Doesn't this also not allow people to test ailment thresholds against non-map bosses.  It's a rare use case, so it's not a big deal, but it seems easy enough to add here